src/common/engine.ts

```diff
@@ -108,7 +108,7 @@ export class CurrencyEngine<
     this.txIdMap = {}
     this.txIdList = {}
     this.walletInfo = walletInfo
-    this.walletId = walletInfo.id != null ? `${walletInfo.id} - ` : ''
+    this.walletId = walletInfo.id ?? ''
```

Comment:
The `?? ''` part isn't even needed, since `walletInfo.id` is non-optional, and the core can't live without it.